GPSTest/src/main/java/com/android/gpstest/GpsTestActivity.java

```diff
@@ -439,11 +443,11 @@ private void init() {
             recreate();
         }
 
-        addStatusListener();
-
-        addOrientationSensorListener();
-
-        addNmeaListener();
+//        addStatusListener();
+//
+//        addOrientationSensorListener();
+//
+//        addNmeaListener();
```

Comment:
Do we still need this commented out code?

Reply:
I commented it out to make merges easier when we pull in master. That way we can see that we intentionally commented it out instead of having to figure out where that code went and why master has it and we don't.